e2e-tests/production-runtime/scripts/chunks.js

```diff
@@ -0,0 +1,155 @@
+const fs = require(`fs-extra`)
+const path = require(`path`)
+const yargs = require(`yargs`)
+
+const getAssetManifest = () => {
+  const { assetsByChunkName } = require(`../public/webpack.stats.json`)
+  return assetsByChunkName
+}
+
+const getPagesManifest = () => {
+  const { pages, dataPaths } = require(`../.cache/data.json`)
+
+  const result = {}
```

Comment:
As mentioned in DM, I will be looking in using `cy.server`/`cy.route` to emulate 404 responses for some chunks - if it will prove to be time-sink, then I'll update it with reduce ;)

Reply:
changed in 98a3072